frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -579,7 +836,9 @@ export const getSubscriptionTopologyEdgeItems = (
               data: {
                 resources: {
                   obj: subRes,
-                  connections: [resource, res],
+                  eventSources: getSubscribedEventsources(resource, resources),
+                  channels: [resource],
+                  ksservices: [res],
```

Comment:
same as above. we can just pass these as objects?

Reply:
Using `PubSubResourceOverviewList` to list here as well.